### PR TITLE
📝 add 2 new tutorials: 🙀 OMG there is no class in Golo

### PR DESCRIPTION
**Pull request by k33g**
2 tutorials:
- Structures
- DynamicObjects

Related issue #548

### CODE REVIEW

**Comment by k33g** [comment]
👋 @jponge @yloiseau I'm ready for review

File: tutorials/dynamicobjects.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+```
+
+> - 1️⃣ define a function returning a DynamicObject
+> - 2️⃣ call the function
+> - 3️⃣ it will print `Hello I'm Jane Doe`
+> - 4️⃣ it will print `I'm Jane Doe: 👋 hello world 🌍`
+
+✋ **To run it**, type this command: `golo golo --files 03-dynamicobjects-constructors.golo`
+
+ℹ️ more about sinamic objects: https://golo-lang.org/documentation/next/#_dynamic_objects
```
Comment (yloiseau) [inline tutorials/dynamicobjects.md:124]: `s/sinamic/dynamic/`

**Review by yloiseau — APPROVED** [review]

**Comment by jponge** [comment]
LGTM after fixing the typo found by @yloiseau

File: tutorials/dynamicobjects.md (hunk 1 above)
Comment (k33g) [reply tutorials/dynamicobjects.md]: thanks / done

**Review by jponge — APPROVED** [review]

**Pull request merged**